pages/common/conda.md

```diff
@@ -0,0 +1,35 @@
+# conda
+
+> Package, dependency and environment management for any programming language.
+
+- Create a new environment, installing named packages:
+
+`conda create --name {{environment_name}} {{packages}}`
+
+- List all environments:
+
+`conda info --envs`
+
+- Load or unload an environment:
+
+`source {{activate|deactivate}} {{environment_name}}`
+
+- Delete an environment (remove all packages):
+
+`conda remove --name {{environment_name}} --all`
+
+- Search conda channels for a package by name:
+
+`conda search {{package_name}}`
+
+- Install one or more packages into the current environment:
```

Comment:
There are almost always things that have to be left out when creating a tldr page. The best we can do is combine two or three options in a single example, but that can't make the example too overwhelming or unnecessarily complex. In this case I suspect leaving the notion of channels out is a reasonable compromise.